src/diffusers/models/attention_processor.py

```diff
@@ -36,6 +36,13 @@
 else:
     xformers = None
 
+if is_torch_xla_available():
+    from torch_xla.experimental.custom_kernel import flash_attention
```

Comment:
Does this need to go through any version check guards too i.e., a minimum version known to have `flash_attention`?

Reply:
Introduced the version check function `is_torch_xla_version` in import_utils.py. Added the version check for torch_xla here.